src/app/routes/getInitialData/utils/preprocessorRulesConfig/index.js

```diff
@@ -12,8 +12,10 @@ import applySquashTopstories from '#lib/utilities/preprocessor/rules/topstories'
 import addIdsToItems from '#lib/utilities/preprocessor/rules/addIdsToItems';
 import filterGroupsWithoutStraplines from '#lib/utilities/preprocessor/rules/filterGroupsWithoutStraplines';
 import addSummaryBlock from '#lib/utilities/preprocessor/rules/cpsAssetPage/addSummaryBlock';
+import cpsOnlyOnwardJourneys from '#lib/utilities/preprocessor/rules/cpsAssetPage/cpsOnlyOnwardJourneys';
 
 export const cpsAssetPreprocessorRules = [
+  cpsOnlyOnwardJourneys,
```

Comment:
To DO:
This rule _has to be first_ because logic linked below seems to filter out the `"type": "see-alsos"` field from the groups in related content when it shouldn't:
https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/preprocessor/rules/cpsAssetPage/timestampToMilliseconds/index.js#L37..L54 

I'll fix https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/preprocessor/rules/cpsAssetPage/timestampToMilliseconds/index.js so it doesn't have this unintended side-effect.

This is another argument for https://github.com/bbc/simorgh/pull/5179

Reply:
Addressed by 70419e3ffee9aeefda23baba4b43bf924a25b1f6 and 88ef4a8a9aeb81b661cf7123d680a36ee7893837